internal/pkg/cli/storage_init.go

```diff
@@ -445,7 +551,6 @@ func (o *initStorageOpts) validateWorkloadName() error {
 }
 
 func (o *initStorageOpts) Execute() error {
-
 	return o.createAddon()
```

Comment:
Maybe we should remove this function since it doesn't add any value at all...

Reply:
Updated! I think it does add some semantic value to explain what's being done in `Execute`... To leave it or not doesn't make so much difference.